incubator/kubernetes-vault/values.yaml

```diff
@@ -5,21 +5,35 @@
 replicaCount: 3
 image:
   repository: boostport/kubernetes-vault
-  tag: v0.4.8
+  tag: v0.5.2
   pullPolicy: Always
 app: kubernetes-vault
 service:
   dummyPort: 80
-vault:
-  address: http://vault:8200
-  token: change-this-value
-  vaultCertBackend: intermediate-ca
-  vaultCertRole: kubernetes-vault
+
+# Maps directly to https://github.com/Boostport/kubernetes-vault#configuration
+configMap:
+  vault:
+    addr: http://vault:8200
+    token: change-this-value
+  kubernetes:
+    service: "{{ template \"fullname\" . }}"
+    watchNamespace: ${KUBERNETES_NAMESPACE}
+    serviceNamespace: ${KUBERNETES_NAMESPACE}
+
+rbac:
+  create: true
+  # apiVersion to use, for example: v1, v1beta1, v1beta2
+  apiVersion: v1
```

Comment:
You shouldn't need to make this configurable.

Reply:
so i should just use v1? What about k8s versions compatiblity?

Reply:
v1 has been available since 1.8. We officially support the current and previous minor version of Kubernetes. Don you need to support older versions?

Reply:
Not really, but wasn't sure what is the policy, gonna remove it now

Reply:
Gonna change deployment to `apps/v1` too i guess